Packages/io.chainsafe.web3-unity/Editor/ChainInfoModel.cs

```diff
@@ -0,0 +1,63 @@
+using System.Collections.Generic;
+
+namespace ChainSafe.Gaming.UnityPackage.Model
```

Comment:
3 suggestions for this class
1. some of these can be structs except the main class
2. if we're deserializing the response we can just have the fields we need (chain name and id)
3. Model in namespace should be replaced with editor or move script into Runtime/Scripts/Model like all other models

Reply:
done